app/src/main/java/fr/free/nrw/commons/upload/worker/UploadWorker.kt

```diff
@@ -532,33 +533,33 @@ class UploadWorker(
     }
 
     private fun findUniqueFileName(fileName: String): String {
-        var sequenceFileName: String?
-        var sequenceNumber = 1
-        while (true) {
+        var sequenceFileName: String? = fileName
+        val chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789"
```

Comment:
In order to avoid accidentally forming words or strings that look like codenames, would you mind restricting to **digits**? 5 digits is already vastly enough, as I don't think any caption has ever exceeded 200 occurences. Thanks!

Reply:
No worries. I changed to a 5-digit numeric hash and have updated the PR description accordingly.

Reply:
Is `chars` still needed?